lib/srv/sess.go

```diff
@@ -922,6 +928,9 @@ func (s *session) emitSessionJoinEvent(ctx *ServerContext) {
 	}
 
 	// Emit session join event to Audit Log.
```

Comment:
The comment now refers to the line below. I would move it down, or just remove it.